crates/accelerate/src/sparse_pauli_op.rs

```diff
@@ -298,7 +299,7 @@ impl MatrixCompressedPaulis {
     /// explicitly stored operations, if there are duplicates.  After the summation, any terms that
     /// have become zero are dropped.
     pub fn combine(&mut self) {
-        let mut hash_table = HashMap::<(u64, u64), Complex64>::with_capacity(self.coeffs.len());
+        let mut hash_table = IndexMap::<(u64, u64), Complex64>::with_capacity(self.coeffs.len());
```

Comment:
Do you want to use ahash for the hasher here so the lookup performance is basically the same as `HashMap`?

Reply:
It should have no meaningful effect on the whole algorithm's performance because this bit is immeasurable compared to the cost of constructing the matrix, but the only reason I didn't do it is because I forgot.

My kids are awake now, so if somebody wanted to push up that change in time to get this into 1.3.0, it's fine by me - I won't have time til tomorrow.

Reply:
Done in 23ba5f4 🙂